src/components/tabs/_tabs.scss

```diff
@@ -239,7 +239,7 @@
     outline: 2px solid transparent;
     border-bottom: 1px solid $ui-04;
     background-color: $field-01;
-    @include breakpoint(bp--sm--major) {
+    @include carbon--breakpoint(md) {
```

Comment:
it seems at this breakpoint we lose display of the tabs altogether

![image](https://user-images.githubusercontent.com/8265238/54549523-35dba100-4978-11e9-86a2-3d87f6419e8a.png)

this is a bug that can also be seen in `master`, except that it occurs at 768px width instead

![image](https://user-images.githubusercontent.com/8265238/54549589-5f94c800-4978-11e9-8dfe-6422c3bdeedc.png)

I'm not sure if this issue is tracked elsewhere, but should we address it in a separate ticket?

Reply:
Thanks @emyarod for pointing it out! If it happens on `master` we should track it in a separate ticket.